src/ipc/ipc4/helper.c

```diff
@@ -382,6 +379,10 @@ int ipc_comp_connect(struct ipc *ipc, ipc_pipe_comp_connect *_connect)
 		goto e_sink_connect;
 	}
 
+	ret = comp_bind(source, bu);
+		if (ret < 0)
```

Comment:
you'd also need to fix the error paths below. e_sink_connect label no longer needs to do an unbind right?

Reply:
Some whitespace error on L383 and L384, indentation seems off.

Reply:
whitespaces fixed